pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/nonpepperkit/KitResponse.java

```diff
@@ -1,10 +1,5 @@
 package org.broadinstitute.dsm.model.nonpepperkit;
 
-import lombok.AllArgsConstructor;
-
-@AllArgsConstructor
 public class KitResponse {
-    public String errorMessage;
-    public String juniperKitId;
-    public Object value;
+
```

Comment:
Does this class simply exist so callers can distinguish an error response from a non-error response?
I have not been a fan of this pattern, but now that you refactored it feels like a simple boolean `isError` flag on KitResponse<Error> would be more straightforward?